src/util/codec/datum.rs

```diff
@@ -353,7 +355,13 @@ mod test {
     fn test_datum_cmp() {
         let tests = vec![
             (Datum::F64(1.0), Datum::F64(1.0), Ordering::Equal),
+            (Datum::F64(1.0), Datum::F64(f64::MAX), Ordering::Less),
+            (Datum::F64(1.0), Datum::F64(f64::MIN), Ordering::Greater),
+            (Datum::F64(1.0), Datum::F64(f64::INFINITY), Ordering::Less),
```

Comment:
Yes, go has.
I see the expr like `a > 1` then the range is `(1, +inf)`, so maybe we should support it later.
@coocood Am i right?